doc/conf.py

```diff
@@ -78,3 +78,8 @@
     "github_url": "https://github.com/exasol/python-toolbox",
     "accent_color": "grass",
 }
+# -- Configure link checking behavior  ----------------------------------------
+extra_linkcheck_ignores = os.getenv("SPHINX_EXTRA_LINKCHECK_IGNORES")
```

Comment:
Depending on what is decided with the https, I think that we could remove this SPHINX_EXTRA_LINKCHECK_IGNORES check. As each repo will have it's own conf.py, it should be ok for it to be added to manually as well in the `linkcheck_ignore`